Buildings/Resources/ReferenceResults/Dymola/Buildings_Experimental_DHC_EnergyTransferStations_Combined_Generation5_Examples_ChillerBorefield.txt

```diff
@@ -1,4 +1,4 @@
-last-generated=2021-05-01
```

Comment:
What is the justification for the small differences here? (same for the other example/validation models that have different results beyond just the variable name change). Is this from the changes to the heat exchanger?

Reply:
This is mainly due to the changes in `Buildings.Experimental.DHC.EnergyTransferStations.Combined.Generation5.Controls.HeatExchanger`.
Previously the primary pump speed (or valve opening) was modulated with a PI loop controlling the temperature difference across the primary side of the heat exchanger. Running more realistic simulations with multiple instances of the model, that control logic turned out to be deficient. It was changed to a simpler a more robust logic where the primary side is controlled based on the demand for heat/cold rejection, so using the same signal as the secondary side. In addition a pressure independent valve is used so that the ETS does not require more primary flow rate than design (this also addresses the hydronic balancing issues that were observed).